src/python/WMCore/Services/pycurl_manager.py

```diff
@@ -162,6 +219,7 @@ def encode_params(self, params, verb, doseq, encode):
 
         return encoded_data
 
+    @portForward(8443)
     def set_opts(self, curl, url, params, headers,
```

Comment:
this method signature might be a problem if we follow the decorator option discussed on the call.

Reply:
I just checked, and it seems to me we do not even need to decorate this one, since we are about to do so for the only two methods that  are calling it (from the current class) [1]. And I did not find any external links to it so far.


[1]
https://github.com/dmwm/WMCore/blob/2b737f326694ae57da72543fb733a1aa7c957177/src/python/WMCore/Services/pycurl_manager.py#L277
https://github.com/dmwm/WMCore/blob/2b737f326694ae57da72543fb733a1aa7c957177/src/python/WMCore/Services/pycurl_manager.py#L324